osu.Game.Rulesets.Osu.Tests/TestSceneSpinner.cs

```diff
@@ -75,11 +75,7 @@ public TestDrawableSpinner(Spinner s, bool auto)
             protected override void CheckForResult(bool userTriggered, double timeOffset)
             {
                 if (auto && !userTriggered && Time.Current > Spinner.StartTime + Spinner.Duration / 2 && Progress < 1)
-                {
-                    // force completion only once to not break human interaction
-                    Disc.RotationAbsolute = Spinner.SpinsRequired * 360;
-                    auto = false;
-                }
+                    Disc.RotationAbsolute += (float)(Spinner.Duration / 120);
```

Comment:
where did this number come from?

Reply:
just a random number to make the test spinner spin smoothly, you can change that if you have a better idea.

Reply:
no worries, just wanted to check.